modmail/utils/pagination.py

```diff
@@ -57,13 +59,14 @@ def __init__(
         contents: Union[List[str], str],
         /,
         source_message: Optional[discord.Message] = None,
-        embed: Embed = None,
+        embed: Optional[Embed] = _AUTOGENERATE,
```

Comment:
Add a line about passing `None` for no embeds in the docstring.

Reply:
170fa3a